src/test/java/fr/univartois/sonargo/AbstractSonarTest.java

```diff
@@ -28,6 +26,12 @@ public void init(DefaultFileSystem fileSystem) {
 		GoProperties.JUNIT_REPORT_PATH_DEFAULT);
 
 	testerContext.settings().appendProperty(CoreProperties.PROJECT_EXCLUSIONS_PROPERTY, "vendor/**,.git/**");
+	testerContext.settings().appendProperty(CoreProperties.PROJECT_TEST_INCLUSIONS_PROPERTY, "**/**_test.go");
+	testerContext.settings().appendProperty(CoreProperties.PROJECT_INCLUSIONS_PROPERTY, "**/**.go");
+	testerContext.settings().appendProperty("sonar.sources", "./");
```

Comment:
Isn't it necessary to exclude from the source files tests files? Or are files found both in the source path and the test path always considered as tests?